pandas/tests/io/test_pytables.py

```diff
@@ -736,6 +736,20 @@ def test_put_compression_blosc(self):
             store.put('c', df, format='table', complib='blosc')
             tm.assert_frame_equal(store['c'], df)
 
+    def test_complibs(self):
+        df = tm.makeDataFrame()
+
```

Comment:
How do I best do that? There is the h5dump utility that can tell me if its compressed or I can compare the size of the files perhaps?